library/tendrl_user.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/python
+'''
+module: test_user
+short_description: Changes 'usm_username' user password to _password_from_module_argument_
+via Tendrl API.
+description:
+    - Module reads from confile(module argument 'conf_path') 'usm_username',
+      'usm_password' and 'usm_api_url'.
+    - There must be installed tendrl-api on the machine where this is ran.
+    - Usage: ansible -i _inventory_ -m tendrl_user -a "conf_path='_usm.ini.path_'" localhost
+version_added: "1.0"
+author: "fbalak@redhat.com", "mkudlej@redhat.com"
+'''
+
+import json
+import ConfigParser
+import requests
+from ansible.module_utils.basic import AnsibleModule
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
```

Comment:
Python imports are placed after ansible special variables according to http://docs.ansible.com/ansible/dev_guide/developing_modules_documenting.html.